app/scripts/components/common/layout-root/index.tsx

```diff
@@ -56,6 +56,8 @@ function LayoutRoot(props: { children?: ReactNode }) {
   const { children } = props;
   const [sessionStart, setSesstionStart] = useState<string | undefined>();
   const sessionItem = window.sessionStorage.getItem(SESSION_KEY);
+  const { pathname } = useLocation();
```

Comment:
Im a fan of using `useLocation` but want to double check that this isn't going to cause a problem when implemented in NEXT.js?

Reply:
yes, thanks for checking. We are not going to export layout root for library. This is where things that are unique to instances should happen.!